airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -229,32 +231,35 @@ export const getInitialTransformations = (operations: OperationCreate[]): Operat
 
 export const getInitialNormalization = (
   operations?: Array<OperationRead | OperationCreate>,
-  isEditMode?: boolean
+  isNotCreateMode?: boolean
 ): NormalizationType => {
   const initialNormalization =
     operations?.find(isNormalizationTransformation)?.operatorConfiguration?.normalization?.option;
 
   return initialNormalization
     ? NormalizationType[initialNormalization]
-    : isEditMode
+    : isNotCreateMode
     ? NormalizationType.raw
     : NormalizationType.basic;
 };
 
 export const useInitialValues = (
   connection: ConnectionOrPartialConnection,
   destDefinition: DestinationDefinitionSpecificationRead,
-  isEditMode?: boolean
+  isNotCreateMode?: boolean
```

Comment:
Negated boolean values are harder to read. Could we make this `isCreateMode` instead and just flip the checks that are using this value, please.